src/main/java/Duke.java

```diff
@@ -6,5 +10,81 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        Scanner in = new Scanner(System.in);
+
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        String input = in.nextLine();
+        Task[] list = new Task[100];
```

Comment:
Any purpose of using 100? Would be could if you could declare as constant with its purpose